trafficjam-functions.sh

```diff
@@ -191,10 +191,15 @@ function get_local_load_balancer_ip() {
 }
 
 function iptables_tj() {
+	if [ -z "$(iptables -L | grep DOCKER)" ]; then
```

Comment:
We should do this check once on startup. For some reason calling iptables from within the container is horrendously slow (still haven't figured out why), so this would essentially slow every call by 2X.

Can you add a function in `trafficjam-functions.sh` like `detect_iptables_version()` that does the check and sets a var like `IPTABLES_CMD`?

Then we can call it once right at the beginning of `trafficjam.sh` (L59): https://github.com/kaysond/trafficjam/blob/84348e22a0624259f8e4fc6fc3aed32064a20afd/trafficjam.sh#L57-L66

Reply:
We should also make the check more robust by listing that chain specifically via iptables args `if iptables-nft -L DOCKER-USER &> /dev/null; then`